SofaKernel/modules/SofaCore/src/sofa/core/behavior/MultiMatrix.h

```diff
@@ -65,8 +65,13 @@ class MultiMatrix
         parent->m_resetSystem();
     }
 
-    /// m = m*M+b*B+k*K
+    [[deprecated("Use setSystemMBKMatrix instead.")]]
```

Comment:
Please use the macro SOFA_ATTRIBUTE_DEPRECATED. It is meant to set a deprecation period and thus a date for future removal.

Reply:
Done. Thank you @guparan